gnome-initial-setup/pages/timezone/gis-timezone-page.c

```diff
@@ -469,15 +508,6 @@ gis_timezone_page_locale_changed (GisPage *page)
   gis_page_set_title (GIS_PAGE (page), _("Time Zone"));
 }
 
-static void
-gis_timezone_page_shown (GisPage *page)
```

Comment:
I assume that the point of this was to not modify the timezone while the user is trying to set it manually. What should happen in the following sequence of events?

* Network connection comes up
* This page asks Geoclue for location
* User advances to this page, and manually chooses location A
* Geoclue replies with location B

Should we overwrite the user's choice? I think that is what will happen with this patch.

Reply:
Yeah, as explained in the commit message, this cannot happen. The implementation cancels any ongoing geoclue requests if a location is set (and a location can be set by the user, so, if they are quick enough, it'll be the users' location to be used and they will never get it reassigned).